spec/system/candidate_interface/entering_details/international_candidate_enters_their_other_qualification_spec.rb

```diff
@@ -46,9 +51,9 @@
 
     when_i_click_add_another_qualification
     and_i_select_add_other_non_uk_qualification
-    and_i_fill_in_the_name_of_my_qualification
+    and_i_fill_in_the_name_of_my_new_qualification
```

Comment:
Is this necessary? It looks like we sufficiently exercise the behaviour fixed by the bug by just adding a 2nd "Master Rules" qual.

Reply:
Actually just realised what this might be about — should we be adding an Other UK qual, rather than non-uk?